### PR TITLE
Fixes softcrit

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
fix: You can no longer drag people to death who are in crit.
/:cl:

### CODE REVIEW

File: hippiestation/code/modules/mob/mob_movement.dm [hunk 1]
```diff
@@ -50,3 +50,11 @@ proc/Can_ShadowWalk(var/mob/mob)
...
 
 	return FALSE
+
+/mob/living/carbon/Move(atom/newloc, direct)
+	. = ..(newloc, direct)
+	if(lying && !pulledby)
+		if(stat == SOFT_CRIT)
+			visible_message("<span class='danger'>[src] painfully crawls forward!</span>", "<span class='userdanger'>You crawl forward at the expense of some of your strength.</span>")
+			apply_damage(1, OXY)
+		playsound(src, pick('hippiestation/sound/effects/bodyscrape-01.ogg', 'hippiestation/sound/effects/bodyscrape-02.ogg'), 20, 1, -4)
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/mob_movement.dm]: Am I crazy or should this be indented?

Comment (ghost) [reply hippiestation/code/modules/mob/mob_movement.dm]: most likely it should be indented

Comment (tortellinitony) [reply hippiestation/code/modules/mob/mob_movement.dm]: no

Comment (KayeArray) [reply hippiestation/code/modules/mob/mob_movement.dm]: explain

Comment (GuyonBroadway) [reply hippiestation/code/modules/mob/mob_movement.dm]: Won't this spam the crawl noise of someone moves while lying down say via a roller bed and fire extinguisher or conveyor belts.

Comment (tortellinitony) [reply hippiestation/code/modules/mob/mob_movement.dm]: yes I have to add a check for buckled and missing limbs..

**Comment by tortellinitony** [comment]
Should be OK now?

**Comment by GuyonBroadway** [comment]
dis looks much better.

**Review by GuyonBroadway — APPROVED** [review]

**Review by j-mie — APPROVED** [review]

**Pull request merged**